dev/PushNotifications/PushNotificationUtility.h

```diff
@@ -8,11 +8,12 @@
 #include "../Common/AppModel.Identity.h"
 #include "wil/stl.h"
 #include "wil/win32_helpers.h"
-#include "PushBackgroundTaskInstance.h"
+#include "MockBackgroundTaskInstance.h"
```

Comment:
This feels strange. I wouldn't expect mocks to appear outside of test directories.

Reply:
Yeah that is the first thing I thought when I saw the "mock" naming convention.

Reply:
Rename to LRPBackgroundTaskInstance

Reply:
Works for me. iThe new name doesn't trigger me anymore :)

thanks.